src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -8,8 +8,22 @@
   $field['init_rows'] = $field['init_rows'] ?? $field['min_rows'] ?? 0;
   $field['max_rows'] = $field['max_rows'] ?? 0;
   $field['min_rows'] =  $field['min_rows'] ?? 0;
-  $field['reorder'] = $field['reorder'] ?? true;
   $field['subfields'] = $field['subfields'] ?? $field['fields'] ?? [];
+  $field['reorder'] = $field['reorder'] ?? true;
+
+  if($field['reorder'] !== false) {
+     switch(gettype($field['reorder'])) {
+         case 'string': {
+            usort($field['value'], fn($a, $b) => $a[$field['reorder']] <=> $b[$field['reorder']]);
+         }
+         break;
+         case 'array': {
+            usort($field['value'], fn($a, $b) => $a[$field['reorder']['name']] <=> $b[$field['reorder']['name']]);
+         }
+         break;
+     }
+  }
```

Comment:
Nice solution reordering in PHP on initial load! Much cleaner than in JS. But this bit threw a huge error for me:

![Screenshot 2022-01-15 at 15 40 36](https://user-images.githubusercontent.com/1032474/149623745-139ead21-ef80-4ad2-b6ed-b7afdfc75b6b.png)

I have `order` and `role` columns on `owner_pet`. In PetCrudController I've added:
```php
        CRUD::field('owners')->subfields([
            ['name' => 'role', 'type' => 'text']
        ])->reorder('order');
```